lib/MusicBrainz/Server/Controller/EditNote.pm

```diff
@@ -0,0 +1,208 @@
+package MusicBrainz::Server::Controller::EditNote;
+use Moose;
+use MooseX::MethodAttributes;
+use namespace::autoclean;
+
+use List::AllUtils qw( first );
+use MusicBrainz::Server::Constants qw( $CONTACT_URL );
+use MusicBrainz::Server::Data::Utils qw( load_everything_for_edits );
+use MusicBrainz::Server::Validation qw( is_database_row_id );
+use MusicBrainz::Server::Translation qw( l );
+
+extends 'MusicBrainz::Server::Controller';
+with 'MusicBrainz::Server::Controller::Role::Load' => {
+    model => 'EditNote',
+    entity_name => 'edit_note',
+};
+
+sub base : Chained('/') PathPart('edit-note') CaptureArgs(0) { }
+
+sub _load
+{
+    my ($self, $c, $edit_note_id) = @_;
+    return unless is_database_row_id($edit_note_id);
+    my $edit_note = $c->model('EditNote')->get_by_id($edit_note_id);
+    $c->model('Editor')->load($edit_note);
+    $c->model('EditNoteChange')->load_latest($edit_note);
+    return $edit_note;
+}
+
+sub detach_if_cannot_modify {
+    my ($self, $c, $edit_note, $edit) = @_;
+
+    my $is_own_note = $c->user && $c->user->id == $edit_note->{editor_id};
+    my $is_admin = $c->user && $c->user->is_account_admin;
+
+    unless ($is_own_note || $is_admin) {
+      $c->stash(
+          component_path  => 'user/UserMessage',
+          component_props => {
+              title    => l('Cannot modify edit note'),
+              message  => l('You cannot modify other users’ edit notes'),
```

Comment:
(Mainly added the period, but another message below uses `can’t`, so suggesting that too for consistency.)
```suggestion
              message  => l('You can’t modify other users’ edit notes.'),
```